### PR TITLE
Add Business model and api endpoints for business

### PR DESCRIPTION
**Pull request by Neeraj-Ghodla**

### CODE REVIEW

File: src/api/accounts/businesses.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+
+{% h4 Optional Fields %}
+
+|   Field   |  Type  |       Description        |
+| :-------- | :----- | :----------------------- |
+| accountId | String | The Centrapay accountId. |
+
+{% warning If accountId is not provided when creating a business, then a new
+account will be created and associated to the business. %}
```
Comment (heymatthew) [inline src/api/accounts/businesses.md:62]: This doesn't seem like a warning. Could we make it a paragraph on it's own?

File: src/api/accounts/businesses.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
```
Comment (heymatthew) [inline src/api/accounts/businesses.md:63]: I think we need some Errors here when:

* An accountId isn't an org account
* The NZBN cannot be found

Comment (Neeraj-Ghodla) [reply src/api/accounts/businesses.md]: Added errors for both cases :)

File: src/api/accounts/businesses.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+  "createdBy": "crn:WIj211vFs9cNACwBb04vQw:api-key:MyApiKey",
+  "updatedBy": "crn:WIj211vFs9cNACwBb04vQw:api-key:MyApiKey",
+}
+```
+
+{% h4 Error Responses %}
+
+| Status |         Code          |                            Description                            |
+| :----- | :-------------------- | :---------------------------------------------------------------- |
+| 403    | ACCOUNT_NOT_SUPPORTED | The Centrapay account provided is not supported for this request. |
```
Comment (ncjones) [inline src/api/accounts/businesses.md:85]: `INVALID_ACCOUNT` : Account does not exist, is not authorized or is the wrong type.

Comment (Neeraj-Ghodla) [reply src/api/accounts/businesses.md]: Changed it :)

**Review by ncjones — APPROVED** [review]

**Pull request merged**